packages/enhanced-img/src/index.js

```diff
@@ -0,0 +1,128 @@
+import path from 'node:path';
+import { image } from './preprocessor.js';
+
+/**
+ * @returns {Promise<import('vite').Plugin[]>}
+ */
+export async function enhancedImages() {
+	const imagetools_plugin = await imagetools();
+	if (!imagetools_plugin) {
+		console.error(
+			'@sveltejs/enhanced-img: vite-imagetools is not installed. Skipping build-time optimizations'
+		);
+	}
+	return imagetools_plugin && !process.versions.webcontainer
+		? [image_plugin(), imagetools_plugin]
+		: [];
+}
+
+/**
+ * Creates the Svelte image plugin which provides the preprocessor.
+ * @returns {import('vite').Plugin}
+ */
+function image_plugin() {
+	const preprocessor = image();
+
+	return {
+		name: 'vite-plugin-enhanced-img',
```

Comment:
should we namespace this, or is that paranoid?

```suggestion
		name: 'vite-plugin-svelte-enhanced-img',
```

Reply:
it just seems so long at that point that I'd rather not